resources/js/Components/Form/CharacterCreateForm.test.js

```diff
@@ -9,7 +8,7 @@ const factions = [
     'The Harpers',
     'The Order of the Gauntlet',
     'The Emerald Enclave',
-    'The Lords\' Alliance',
+    'The Lords Alliance',
```

Comment:
> that apostrophe wasn't there by accident, the name is supposed to be "The Lords' Alliance"

This does not really matter, it's a mocked prop, the reason why it was removed is because eslint does not appreciate having single quote escaped within single quote, fixed for now but this does not affect anything including the test case.